app/src/main/java/com/iitp/njack/iitp_connect/core/youtube/PlaylistAdapter.java

```diff
@@ -0,0 +1,68 @@
+package com.iitp.njack.iitp_connect.core.youtube;
+
+import android.databinding.DataBindingUtil;
+import android.databinding.ViewDataBinding;
+import android.support.annotation.LayoutRes;
+import android.support.annotation.NonNull;
+import android.support.v7.widget.RecyclerView;
+import android.view.LayoutInflater;
+import android.view.ViewGroup;
+import com.iitp.njack.iitp_connect.BR;
+import java.util.List;
+
+public class PlaylistAdapter extends RecyclerView.Adapter<PlaylistAdapter.PlaylistViewHolder> {
+    List<YoutubePlaylist> playlists;
+    PlaylistViewModel playlistViewModel;
+    private int layoutId;
+
+    public PlaylistAdapter(@LayoutRes int layoutId, PlaylistViewModel playlistViewModel) {
+        this.layoutId = layoutId;
+        this.playlistViewModel = playlistViewModel;
+    }
+
+    private int getLayoutIdForPosition(int position) {
+        return layoutId;
+    }
+
+    public void setPlaylists(List<YoutubePlaylist> playlists) {
+        this.playlists = playlists;
```

Comment:
Can you kindly check out the coding calendar adapter and see how I have done it ?